src/test/java/core/basesyntax/strategy/operationhandler/impl/SupplyOperationHandlerTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.strategy.operationhandler.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class SupplyOperationHandlerTest {
```

Comment:
(suggestion only) You also can add test in which supply quantity is 0, because then this operation is redundant, for me) It also relates to ReturnOperationHandlerTest.

Reply:
I don't know why quantity is not ok to be 0, because then you don't do anything with it. But maybe I could have added check in FruitTransactionParseService, so no transaction with int quantity = 0 will even be parsed. Is it OK?